site.yml

```diff
@@ -187,7 +187,6 @@
 
 - name: ceph osds
   hosts: ceph_osds
-  serial: 1
```

Comment:
right. There are many cases I will have to cover. But I would like to do full case tests after optimization.
Currently, I only cover the base cases.